hbase-server/src/main/java/org/apache/hadoop/hbase/util/FSTableDescriptors.java

```diff
@@ -139,6 +140,31 @@ public static TableDescriptor tryUpdateAndGetMetaTableDescriptor(Configuration c
     }
   }
 
+  public static ColumnFamilyDescriptor getTableFamilyDesc(
```

Comment:
Let me fix the name. Reg the class to keep it, maybe it is fine here because it is primarily used by FSTD only. Just for upgrade case, master needs it. But I am open to keeping it in other util class if there is better suggestion.